### ISSUE TITLE
Error 3015004: The type defined in the ABI is invalid

### ISSUE THREAD

**Issue by bensig** [EOSIO/eos#5909]
Please include the name of the type that is invalid in the error reporting.

I am working on a pretty simple contract here. 

I had the wrong type defined as uint64_t instead of uint64. Got this error - Error 3015004 - when I tried to run an action on that contract.

The type defined in the ABI is invalid - but it would be VERY helpful to show the NAME of the TYPE  that was invalid.

**Comment by taokayan** [comment]
Did you use eosio-cpp from cdt or the old eosiocpp to generate abi file? uint64 will be automatically generated by theses tools.

**Comment by jgiszczak** [comment]
There were defects in the processing of exception messages.  The code was attempting to include the name of the type, but it was not processed correctly.  These defects have now been corrected in the v1.3.2 release, available today.

Additionally, the same corrections are queued up for the v1.4 release.

**Comment by flyer88** [comment]
i have same problem
 my `contract.cpp`
```
         [[eosio::action("binddapp")]] 
         void binddapp(std::string &dapp,uint64_t contract_name)
```
`aib file`

```
...
...
"structs": [
        {
            "name": "binddapp",
            "base": "",
            "fields": [
                {
                    "name": "dapp",
                    "type": "string"
                },
                {
                    "name": "contract_name",
                    "type": "uint64"
                }
            ]
        }
...
...
```
i push action by cleos
```
cleos push action tester binddapp '["dappname","user"]' -p user@active
```

how can i fix the problem?
